lib/db/schema/patch-085-086.sql

```diff
@@ -0,0 +1,45 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('85');
+
+ALTER TABLE recoveryKeys ADD COLUMN recoveryKeyIdHash BINARY(32),
+ALGORITHM = INPLACE, LOCK = NONE;
```

Comment:
This table will still have a `recoveryKeyId` column, right?  We'll need a followup bug to drop that column in a subsequent migration.

If we're assuming that the `recoveryKeys` table is empty, it may be worth adding some logic here in the migration to ensure that - either by explicitly deleting any existing rows, or by erroring out if there are rows present.

Reply:
> This table will still have a recoveryKeyId column, right?

It would but I guess since I would be deleting existing rows, I could remove the column in this migration as well.